lib/usd/ui/layerEditor/layerTreeModel.cpp

```diff
@@ -19,6 +19,7 @@
 #include "componentSaveDialog.h"
 #include "layerEditorWidget.h"
 #include "layerTreeItem.h"
+#include "mayaSessionState.h"
```

Comment:
I'm not sure that the LayerEditor tree stuff should be using the mayaSessionState directly. That will make this code non-portable to Max. @pierrebai-adsk what do you think?